### PR TITLE
Refactor/reactivesearch method changes

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type** Refactor

**Description** 
- Removed def for `getQuerySuggestions()`. 
- Changed name for `reactiveSearchv3()` to `reactiveSearch()`.
- Fixed failing test cases.

**Docs PR** https://github.com/appbaseio/Docs/pull/237

### CODE REVIEW

File: src/core/api/reactiveSearchApi.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ import { removeUndefined, validateRSQuery } from '../../utils/index';
  * @param {boolean} settings.enableQueryRules
  * @param {boolean} settings.customEvents
  */
-function reactiveSearchv3Api(query, settings) {
-    const parsedSettings = removeUndefined(settings);
+function reactiveSearchApi(query, settings) {
```
Comment (bietkul) [inline src/core/api/reactiveSearchApi.js:12]: @mohdashraf010897 It seems you missed to change the URL for `reactivesearch` API call.
https://github.com/appbaseio/appbase-js/blob/dea149540016f774c466c5ce0de809e5fbe33132/src/core/api/reactiveSearchApi.js#L29
It should be just `_recativesearch` for new method.

File: src/index.js [hunk 2]
```diff
@@ -7,47 +7,49 @@ import deleteApi from './core/api/delete';
...
-	AppBaseClient.prototype.search = searchApi;
+  AppBaseClient.prototype.search = searchApi;
 
-	AppBaseClient.prototype.msearch = msearchApi;
+  AppBaseClient.prototype.msearch = msearchApi;
 
-	AppBaseClient.prototype.reactiveSearchv3 = reactiveSearchv3Api;
+  AppBaseClient.prototype.reactiveSearch = reactiveSearchApi;
 
-	AppBaseClient.prototype.getQuerySuggestions = getSuggestionsv3Api;
```
Comment (bietkul) [inline src/index.js:36]: @mohdashraf010897 Why did you remove this method? We use it in ReactiveSearch library. As of now, We don't need any change in this method.

**Review by bietkul — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (mohdashraf010897) [reply src/index.js:36]: 👍

**Review by bietkul — APPROVED** [review]
